src/components/HomepageFeatures/index.tsx

```diff
@@ -78,16 +77,16 @@ export default function HomepageFeatures(): JSX.Element {
               <div className={styles.blueLine}>
                 <div></div>
               </div>
-              <p>Users can access Apache Kvrocks via any Redis client</p>
-              {/* <div className={styles.redisCompatible}>
+              <p>User can use a Redis client to visit Apache Kvrocks.</p>
```

Comment:
I think you need to revert this change, since the previous words is more suitable.

Reply:
Ok, the change here will be reverted in the next pr.

Reply:
reverted.